src/app/app-common/PageSource/include/OpenKneeboard/PlainTextPageSource.h

```diff
@@ -37,9 +37,12 @@ struct DXResources;
 class PlainTextPageSource final : public IPageSource {
  public:
   PlainTextPageSource() = delete;
-  PlainTextPageSource(const DXResources&, std::string_view placeholderText);
+  PlainTextPageSource(
+    const DXResources&,
+    uint32_t fontSize,
+    std::string_view placeholderText);
   virtual ~PlainTextPageSource();
-
+  void ChangeFontSize(uint32_t newFontSize);
```

Comment:
- SetFontSize per previous comment
- Is there a specific reason for `uint32_t`? DirectWrite wants a float; I was going to say 'int is fine given we want pixel-aligned', but given that things like '150% DPI scaling' are a thing, floats are actually useful in practice.